--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -54,7 +54,7 @@
 
 link_libraries(-L.)
 add_executable(ydotoold ${SOURCE_FILES_DAEMON})
-target_link_libraries(ydotoold ydotool_library dl pthread boost_program_options uInputPlus evdevPlus)
+target_link_libraries(ydotoold ydotool_library_static dl pthread boost_program_options uInputPlus evdevPlus -static)
 
 add_executable(ydotool_client ${SOURCE_FILES_CLIENT})
 target_link_libraries(ydotool_client ydotool_library boost_program_options pthread uInputPlus evdevPlus)
@@ -82,4 +82,4 @@
 
 install(TARGETS ydotool_library ydotool_library_static DESTINATION lib)
 #install(TARGETS mousemove key click type recorder DESTINATION lib/ydotool)
-install(TARGETS ydotoold ydotool_client DESTINATION bin)
\ No newline at end of file
+install(TARGETS ydotoold ydotool_client ydotool_client_static DESTINATION bin)
